app/src/main/res/layout-sw600dp/fragment_channel_parameters.xml

```diff
@@ -117,22 +107,27 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toRightOf="@+id/spinner_range_ch2_cp" />
 
-    <CheckBox
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:id="@+id/checkBox_mic_cp"
-        app:layout_constraintLeft_toLeftOf="@+id/checkBox_ch3_cp"
-        app:layout_constraintBottom_toBottomOf="parent"
-        android:layout_marginBottom="16dp" />
-
-    <Spinner
-        android:layout_width="0dp"
-        android:layout_height="48dp"
-        android:id="@+id/spinner_mic_select_cp"
-        app:layout_constraintLeft_toRightOf="@+id/checkBox_mic_cp"
-        app:layout_constraintBottom_toBottomOf="parent"
-        android:layout_marginBottom="8dp" />
-
+        <CheckBox
+            android:id="@+id/built_in_mic_cb"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/built_in_mic_option"
+            android:textSize="18sp"
```

Comment:
Move all the hard-coded dimension values to `dimens.xml`

Reply:
Can we do this later . I know it is important but the layout has a lot of places with hard coded dimensions .. and it would take time to change all of them to dimens.xml . Will it be okay if we do this later. I will try to focus on some other issues related to layouts and UI. Thanks

Reply:
Why did you tick "I have used resources from strings.xml, dimens.xml and colors.xml without hard-coding them" as done? Why do later? Why downgrading the quality of work like this?

Reply:
There are many hard coded dimensions already in the file that weren't added by me. But okay, i will move all of them to dimens.xml. I didn't intend to downgrade work quality.

Reply:
Don't make code for the sake of commit count. Thanks.